### ISSUE TITLE
remove font-awesome CSS import

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#566]
We only use font-awesome (I believe) in the mobile close/open sidebar icons. By using SVG paths (like in static) we save fetching the fa CSS & icon sets

**Comment by jameshadfield** [comment]
Closed in favor of #702

**Comment by jameshadfield** [comment]
With the introduction of `react-icons` in PR #1041 we can now easily include font-awesome icons (as well as others) using syntax such as:
```js
import { FaChevronUp, FaChevronDown } from "react-icons/fa";
```

We still import the Font Awesome CSS (42kb, 7kb gzipped) and use it in the following places (as far as I can tell) for a total of 6 icons:
* https://github.com/nextstrain/auspice/blob/master/src/components/framework/footer.js#L381
* https://github.com/nextstrain/auspice/blob/master/src/components/framework/sidebar-chevron.js#L23
* https://github.com/nextstrain/auspice/blob/master/src/components/framework/sidebar-toggle.js#L47-L55

These should be replaced with imports from `react-icons` & the font-awesome CSS removed from auspice.

**Comment by joachimschmidt557** [comment]
If it's not already being done, I can take care of migrating to `react-icons`.

**Comment by jameshadfield** [comment]
Closed via #1065 & 0c5721e0afd05fa36d5b7aa364d0bd344cfe405e
